front/src/styles/variables.css

```diff
@@ -0,0 +1,28 @@
+.second-color {
+    color: #FFA21C;
+}
+
+.main-color, button.main-color {
+    color: white;
+}
+
+.color-background, button.color-background, ul.color-background {
+    background-color: #6663FF;
+}
+
+button.second-color {
+    color: #6663FF;
+}
+
+.footer-color-background, button.footer-color-background {
+    background-color: #5956FF;
+}
+
+ul.main-color > li.main-color {
+    color: white;
+}
+
+ul.main-color > li.main-color.is-active {
+    border-bottom: 3px solid white;
+    color: #FFA21C;
```

Comment:
em, it is css rules, might they could be ... in common.css
but variables are variables - https://www.w3schools.com/css/css3_variables.asp